Microsoft.Toolkit.Uwp.UI.Controls/RotatorTile/RotatorTile.cs

```diff
@@ -49,6 +49,18 @@ public sealed class RotatorTile : Control
         private WeakEventListener<RotatorTile, object, NotifyCollectionChangedEventArgs> _inccWeakEventListener;
 
         /// <summary>
+        /// Identifies the <see cref="MinRandomDuration"/> property.
+        /// </summary>
+        public static readonly DependencyProperty MinRandomDurationProperty =
+            DependencyProperty.Register(nameof(MinRandomDuration), typeof(int), typeof(RotatorTile), new PropertyMetadata(default(int)));
+
+        /// <summary>
+        /// Identifies the <see cref="MinRandomDuration"/> property.
+        /// </summary>
+        public static readonly DependencyProperty MaxRandomDurationProperty =
```

Comment:
My thinking was you have a property you set for the interval you want, then a second property for extra randomization. Set the latter to TimeSpan.Zero to disable randomization. IMHO this is simpler and clearer than a min/max value, where the max can't be smaller than min, and if you want to disable randomization you have to set max==min